src/main/scala/raster/SparseIntTile.scala

```diff
@@ -0,0 +1,72 @@
+package vectorpipe.raster
```

Comment:
Is this package in use somewhere?  It's a bit weird of a concept in an OSM package...

Reply:
Agreed.

It's used for creating tiles containing density of recent edits, so maybe it belongs on `osmesa.analytics`